test_conformance/api/test_mem_object_info.cpp

```diff
@@ -720,6 +720,8 @@ int test_get_image_info( cl_device_id deviceID, cl_context context, cl_mem_objec
         clReleaseMemObject( imageObject );
     }
 
+    free_mtdata(d);
```

Comment:
An even better solution would be to replace the declaration/initialization of `MTdata d` by `MTdataHolder`, so that no manual `free_mtdata` calls are needed.  That would also fix the leaks on error paths.

Reply:
Thank you for your suggestion. Modified accordingly.